src/config.rs

```diff
@@ -417,8 +418,15 @@ fn config_from_env() -> EnvConfig {
 
     let gcs = env::var("SCCACHE_GCS_BUCKET").ok()
         .map(|bucket| {
+            let url = env::var("SCCACHE_GCS_CREDENTIALS_URL").ok();
             let cred_path = env::var_os("SCCACHE_GCS_KEY_PATH")
                 .map(|p| PathBuf::from(p));
+
+            if url.is_some() && cred_path.is_some() {
+                warn!("Both SCCACHE_GCS_CREDENTIALS_URL and SCCACHE_GCS_KEY_PATH are set");
+                warn!("You should set only one of them!");
```

Comment:
Let's mention that the path will take precedence over the url here.

Reply:
Done